execution/executor/src/chunk_executor.rs

```diff
@@ -306,11 +329,18 @@ impl<V: VMExecutor> ChunkExecutorInner<V> {
             parent_state.current_version,
         );
 
-        // Verify input transaction list.
-        txn_output_list_with_proof.verify(
-            verified_target_li.ledger_info(),
-            Some(first_version_in_request),
-        )?;
+        {
+            let _timer = APTOS_CHUNK_EXECUTOR_OTHER_SECONDS.timer_with(&["apply_chunk__verify"]);
+            // Verify input transaction list.
+            THREAD_MANAGER
```

Comment:
if verify() fails, we don't want to continue the execution. so there is a dependence here instead of parallel ?

Reply:
I have `?` at line 342 to return the error.
(the code here is to set the threadpool, the par_iter is inside the verify function)